shared/src/business/useCases/practitioners/updatePractitionerUserInteractor.js

```diff
@@ -174,8 +174,4 @@ exports.updatePractitionerUserInteractor = async (
     },
     userId: requestUser.userId,
   });
-
-  return new Practitioner(updatedUser, { applicationContext })
```

Comment:
I believe I understand why this is being removed but I am concerned as this is changing currently accepted/deployed functionality. What is the reason for this change?

Reply:
Talked this over in Zoom.